config/initializers/turbo_stream.rb

```diff
@@ -0,0 +1,52 @@
+module TurboStreamDispatchEventHelper
+  # Creates a turbo stream tag like:
+  #   <turbo-stream action="dispatch_event" event="some_event" data="{}"></turbo-stream>
+  #
+  def dispatch_event(event:, data: {})
+    # Allows data to be serialized before dispatching it
+    data = data.to_json unless data.is_a? String
```

Comment:
I was thinking about writing the event data inside the turbo stream (it makes debugs via network tab easier). 

```html
<script type="application/json">
  {
    "title": "...",
  }
</script>
```


And parsing it on the handler:

```javascript

JSON.parse(SCRIPTTAG.textContent);

```

Reply:
I see! Ok, let's do it in some next PR!